src/transformers/linexp_fit.rs

```diff
@@ -0,0 +1,116 @@
+use crate::transformers::transformer::*;
+
+use conv::prelude::*;
+use macro_const::macro_const;
+
+const INPUT_FEATURE_SIZE: usize = 5;
+
+macro_const! {
+    const DOC: &str = r#"
+Transform LinexpFit features to be more usable
+
+The LinexpFit feature extractor returns the following features:
+- amplitude - kept as is
+- fall_slope - kept as is
```

Comment:
I think this descriptions should be changed to something like we have for BazinFit
```
- mag_amplitude - amplitude in fluxes is assumed to be intersect object
  flux, so we transform it to be apparent magnitude: zp - 2.5 log10(2 * A)
- baseline_amplitude_ratio - ratio of baseline to amplitude (both taken
  in original units, not magnitudes, because baseline can be negative)
```